scripts/make-guac.sh

```diff
@@ -226,7 +226,26 @@ write_guacamole_dockerfile()
         printf "CMD [\"/opt/guacamole/bin/start.sh\" ]\n"
     ) > "${guac_dockerfile}"
     log "Successfully added Guacamole Dockerfile"
-}  # ----------  end of function write_brand  ----------
+}  # ----------  end of function write_guacamole_dockerfile  ----------
+
+
+# revert guacd docker to ubunt base image due to freerdp issue in debian base
+# see https://jira.apache.org/jira/browse/GUACAMOLE-707 for more details
+write_guacd_dockerfile()
+{
+    log "Reverting guacd docker to ubunut base"
+    git clone https://github.com/apache/guacamole-server.git "$GUACD_PATH" | log
```

Comment:
We should probably clone from their upstream, where they are now creating the tags, so we can pin to a specified version...

```suggestion
    git clone --branch $GUACD_VERSION https://git-wip-us.apache.org/repos/asf/guacamole-server.git "$GUACD_PATH" | log
```